cluster/examples/kubernetes/ceph/crds.yaml

```diff
@@ -924,10 +924,18 @@ spec:
                       type: object
                       nullable: true
                       properties:
-                        enabled:
```

Comment:
Do we need to keep enabled here on `bucket` for backwards compatibility?

Reply:
Oh. I see this was added incorrectly from the start and should have been `disabled` all along.

Reply:
I see types.go specifies the `HealthCheckSpec` type, which only has `disabled`. This schema just looks incorrect, which would also likely have been caught by schema generation in #7103. So I think we're good to go.